rest/model/distro.go

```diff
@@ -362,7 +361,6 @@ type APIDistro struct {
 	Provider              *string                  `json:"provider"`
 	ProviderSettings      map[string]interface{}   `json:"settings"`
```

Comment:
Does `APIDistro` still need this type?

Reply:
Yes if we want to return an error in case users are relying on the old syntax, which I think we should do.